infra/base-images/base-runner/Dockerfile

```diff
@@ -50,6 +50,9 @@ RUN apt-get update && apt-get install -y \
     wget \
     zip --no-install-recommends
 
+COPY install_perf.sh $WORK
```

Comment:
Can you do this lower down in the file?
Since this step is relatively cheap (i assume) i don't think it's a good idea to put it so high up where it can invalidate more expensive steps like installing go.

Reply:
Done